### ISSUE TITLE
Recollect calendar view, reminders, and printing

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#3399]
### As a [user type] I want [capability] so that [benefit].

**Is your feature request related to a problem? Please describe.**
Keri in ARR made this request re: Recollect data:
> One comment I will make is that I would prefer the collection schedule to show as a calendar (at least on [this page](https://alpha.austin.gov/en/housing-utilities/recycling-trash-and-compost/household-waste/trash-recycling-pickup/)), rather than in a list. The options to sign up for reminders and print the calendar are missing as well, and those are important. We want people to sign up for the reminders because we have the capability to send a short message to them alongside their reminders.

The trash/recycling page says you can sign up for reminder or print out your calendar, but our widget currently doesn't allow it! That's a problem.
https://alpha.austin.gov/en/housing-utilities/recycling-trash-and-compost/household-waste/trash-recycling-pickup/

Here's the full calendar display, we're not using it on alpha.austin.gov currently but could style it as needed.
<img width="595" alt="Screen Shot 2019-10-16 at 9 13 58 AM" src="https://user-images.githubusercontent.com/6384156/66927404-5e527900-eff5-11e9-881b-193bdfe6bb70.png">
<img width="588" alt="Screen Shot 2019-10-16 at 9 14 06 AM" src="https://user-images.githubusercontent.com/6384156/66927412-60b4d300-eff5-11e9-9b45-5e36f5a51664.png">
<img width="592" alt="Screen Shot 2019-10-16 at 9 14 14 AM" src="https://user-images.githubusercontent.com/6384156/66927417-63afc380-eff5-11e9-9217-01f93650cdd3.png">
<img width="593" alt="Screen Shot 2019-10-16 at 9 14 21 AM" src="https://user-images.githubusercontent.com/6384156/66927421-65798700-eff5-11e9-95d1-58b663c9ab25.png">

Here's how recollect displays on austintexas.gov:
https://austintexas.gov/austincomposts
![image](https://user-images.githubusercontent.com/5839508/68687756-48bc6a80-0533-11ea-843d-cb6c05e15c38.png)

**Describe the solution you'd like**
- [ ] Designs for displaying or linking to the full calendar, at least from the [Look up your trash/recycling/compost dates page](https://alpha.austin.gov/en/housing-utilities/recycling-trash-and-compost/household-waste/trash-recycling-pickup/)
- [ ] Designs/copy/acceptance criteria for:
  - [ ] Signing up for reminders
  - [ ] Printing out the calendar

**Describe alternatives you've considered**

**Additional context**

**Comment by desigonz** [comment]
On desktop, could do calendar; on mobile could do our pared down solution.

**Comment by desigonz** [comment]
Goals for sprint 44:
- [ ] @desigonz to share notes
- [ ] talk to @briaguya about what's possible
- [ ] @chasechenevert and @toribr to do some sketches about how this might work

**Comment by desigonz** [comment]
### Notes from ARR
- Calendar view is important to them—want users to have the ability to see a full year/far in advance, not just the current month
- Want residents to be able to print the calendar and post in their homes
- Reminder
  - 40,000 customers (out of 200,000) have signed up to get reminders
  - They had a campaign over the summer to increase reminder sign ups (it was successful—doubled the number of sign ups) and want to keep the momentum going
  - They use the reminder system for other things too, like communicating during emergencies
- Keri to share usage data with @desigonz 


### in other news
- [ ] I'm going to send an email to David Ondich asking for his thoughts on the accessibility of the existing implementation of the "find your collection schedule" app.

**Comment by desigonz** [comment]
Oh, and lastly, Marni mentioned that another direction could be to talk to the creators of Recollect (our contact is David Eaves at Harvard) about some of the design changes we'd be interested in.

**Comment by chasechenevert** [comment]
Latest from @desigonz re: David's testing
>FYI as y'all work on the Recollect design—I asked David to see what the experience of accessing his calendar, signing up for reminders, and print a calendar is like, and it seemed to go well!

**Comment by toribr** [comment]
I think we should actually just add the missing pieces and leave it as is, as it meets accessibility requirements and tweaking the UI is low priority, imo.

**Comment by desigonz** [comment]
I'm cool with that. How do y'all feel about the mobile experience? And can we apply any Janis styling? We'll want acceptance criteria for that.

![image](https://user-images.githubusercontent.com/5839508/69269494-bf421380-0b96-11ea-9d65-b68be50cb4f7.png)

**Comment by chasechenevert** [comment]
Did a quick sweep and pulled out places where I think Janis styling could be applied. There are a few things here that point to hiding or removing things within the UI which would be more complex. 

https://docs.google.com/document/d/1_H6PirVpKL_t5fDWsyf6WfBP0mjex_DtTzgu40wVi_8/edit#

**Comment by desigonz** [comment]
Thanks @chasechenevert! Assigning myself to attempt to turn this into acceptance criteria, but seems like a few things we need to talk through before we can fully solidify the criteria. Maybe we can add this to the agenda for the next design pinup and/or dev-design sync?

**Comment by chasechenevert** [comment]
Yeah I think raising this at both meetings for feedback from each discipline would be helpful. I can mock up a few things as well if that helps us get on the same page (we have all the existing components built in XD we would need to recreate this).

**Comment by desigonz** [comment]
That sounds great, thank you!

**Comment by chasechenevert** [comment]
Here's the idea of where I'm going with this, just note that not all interactions have been fleshed out). The goal is to give the Dev team and stakeholders an idea of how this might looked skinned in Janis. Original screens are below the changes. 

**Some of the changes so far**
- changed the second screen menu items so that they're grouped below the search query that they relate to
- Janis typesetting
- Janis forms styling
- removed duplicate "back" button that appear below body content
- I've made a few microcopy changes for this mockup, but Content would ultimately make and approve these changes

XD Link: https://xd.adobe.com/view/7a35b24b-0703-4330-6c96-3be2d3b45594-0c04/
Proposed changes doc: https://docs.google.com/document/d/1_H6PirVpKL_t5fDWsyf6WfBP0mjex_DtTzgu40wVi_8/edit#

**Comment by chasechenevert** [comment]
## Feedback
**Application banner** 
Should a "Help" button or link be available in the application banner so that it's universally available as it is in the original version?

**Help**
- Are the subheadings more helpful than the original implementation?
- These subheadings are listed alphabetically, but should we use a different order? Considering other languages, this order would obviously change depending on the language. Should we manually list these based on past user behavior, or something else?

**Share > Share with email**
  - "comments" might be too vague in this context.
  - Form is on a page with other sharing options (which also require a second click

**Email reminders**
Selecting day of the week
- Consider toggle, checkbox, or radios
